rating-submit.js

```diff
@@ -0,0 +1,242 @@
+                <span class="close" @click=${this.close}>&times;</span>
+              </div>
+               <h1 id="title">${this.title}</h1>
+               <div class="rating-div">
+                 <rating-element rating="5"></rating-element>
+               </div>
+              <div class="button-div">
+                <button id="submit-button" @click=${this.handleSubmit}>
+                  Submit
+                </button>
+                <button id="cancel-button" @click=${this.close}>Cancel</button>
+              </div>
+            </div>
+          </div>
+          <button class="v-button rating-button" id="rating-button" @click=${this.open}>
+            Rate Me
+          </button>
+          <button class="vl-mobile-rating-button" id="rating-button" @click=${this.open}>
```

Comment:
Please change the id to rating-button-mobile here

Reply:
done